.github/workflows/release.yml

```
@@ -0,0 +1,41 @@
+# Builds the .vsix and attaches it to a GitHub Release. This does NOT publish to
+# the Visual Studio Marketplace — that is a separate, deliberate step
+# (.github/workflows/publish.yml, run manually).
+#
+# Trigger by pushing a tag, e.g. `git tag v0.3.0 && git push origin v0.3.0`,
+# or run manually from the Actions tab (workflow_dispatch).
+name: Release
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)